### PR TITLE
Added Air-Gapped network support in Home and Community screens

### PR DESCRIPTION
**Pull request by A4k9**
Signed-off-by: A4k9 <vansh.a4k9@gmail.com>

## Proposed changes

Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

Added air-gapped network support screens on the Home and Community pages:

Screenshot from 2020-10-08 00-22-53

Screenshot from 2020-10-08 00-22-56

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

File: litmus-portal/frontend/src/components/InfoFilled/index.tsx [hunk 1]
```diff
@@ -79,7 +84,28 @@ const InfoFilledWrap: React.FC = () => {
...
+      {loading ? (
+        <div>
+          <Loader />
+          <Typography>Fetching the data...</Typography>
+        </div>
+      ) : error ? (
+        <div className={classes.errorMessage}>
+          <Center>
+            <Typography variant="h4">
+              It seems you have no internet connection, Please try again When
```
Comment (arkajyotiMukherjee) [inline litmus-portal/frontend/src/components/InfoFilled/index.tsx:99]: can you put this in translation file

File: litmus-portal/frontend/src/pages/Community/index.tsx [hunk 2]
```diff
@@ -27,6 +31,48 @@ const Header2: React.FC = ({ children }) => {
...
+    (state: RootState) => state.communityData
+  );
+
+  if (loading) {
+    return (
+      <Scaffold>
+        <div className={classes.root}>
+          <div>
+            <Typography variant="h3" className={classes.mainHeader}>
+              Community
```
Comment (arkajyotiMukherjee) [inline litmus-portal/frontend/src/pages/Community/index.tsx:44]: same

File: litmus-portal/frontend/src/pages/Community/index.tsx [hunk 3]
```diff
@@ -27,6 +31,48 @@ const Header2: React.FC = ({ children }) => {
...
+            </Typography>
+          </div>
+          <div className={classes.errorMessage}>
+            <Center>
+              <Typography variant="h4">
+                It seems you have no internet connection, Please try again when
```
Comment (arkajyotiMukherjee) [inline litmus-portal/frontend/src/pages/Community/index.tsx:67]: same

File: litmus-portal/frontend/src/pages/Community/index.tsx [hunk 4]
```diff
@@ -27,6 +31,48 @@ const Header2: React.FC = ({ children }) => {
...
+      </Scaffold>
+    );
+  }
+  if (error) {
+    return (
+      <Scaffold>
+        <div className={classes.root}>
+          <div>
+            <Typography variant="h3" className={classes.mainHeader}>
+              Community
```
Comment (arkajyotiMukherjee) [inline litmus-portal/frontend/src/pages/Community/index.tsx:61]: same

**Review by arkajyotiMukherjee — CHANGES_REQUESTED** [review]

**Comment by A4k9** [comment]
@arkajyotiMukherjee, made all the required changes, Please review and confirm

**Review by SarthakJain26 — DISMISSED** [review]

File: litmus-portal/frontend/public/locales/en/translation.yaml [hunk 5]
```diff
@@ -191,3 +194,11 @@ workflows:
   templates: Templates
   analytics: Analytics
   scheduleAWorkflow: Schedule a workflow
+
+#################################################
+############    Air-Gapped Screens    ###########
+#################################################
+
+airGapped:
```
Comment (arkajyotiMukherjee) [inline litmus-portal/frontend/public/locales/en/translation.yaml:202]: give it a more generic name like `noInternet`

**Review by arkajyotiMukherjee — CHANGES_REQUESTED** [review]

File: litmus-portal/frontend/src/views/Community/GeoMap/CountryMap.tsx [hunk 6]
```diff
@@ -8,11 +8,14 @@ import useStyles from './styles';
...
 const CountryMap = () => {
-  const data: string[][] = useSelector(
-    (state: RootState) => state.communityData.google.geoCountry
+  const { communityData } = useSelector(
+    (state: RootState) => state.communityData
   );
   const classes = useStyles();
   const theme = useTheme();
+
+  const data: string[][] = communityData.google.geoCountry;
```
Comment (arkajyotiMukherjee) [inline litmus-portal/frontend/src/views/Community/GeoMap/CountryMap.tsx:17]: the type should be automatically inferred...
`string[][]` is not required

**Review by arkajyotiMukherjee — CHANGES_REQUESTED** [review]

File: litmus-portal/frontend/src/views/Community/GeoMap/CountryMap.tsx (hunk 6 above)
Comment (A4k9) [reply litmus-portal/frontend/src/views/Community/GeoMap/CountryMap.tsx]: done

File: litmus-portal/frontend/public/locales/en/translation.yaml (hunk 5 above)
Comment (A4k9) [reply litmus-portal/frontend/public/locales/en/translation.yaml]: updated

**Review by arkajyotiMukherjee — APPROVED** [review]

**Review by Saranya-jena — APPROVED** [review]

**Pull request merged**